9.4/paper.tex

```diff
@@ -775,19 +769,23 @@ \subsection{Integration with the Computational Geometry Algorithms
   the same mesh has been remeshed.
   \begin{figure}
     \centering
-    \begin{subfigure}[b]{0.4\textwidth}
+    \begin{subfigure}[b]{0.3\textwidth}
       \centering
-      \includegraphics[height=.2\paperheight]{png/intersection_cube_sphere_mesh.png}
+      \includegraphics[width=.2\paperwidth]{png/heart_implicit.png}
+      \caption{\label{fig:heart_tria}}
+    \end{subfigure}
+    \begin{subfigure}[b]{0.3\textwidth}
+      \centering
+      \includegraphics[height=.14\paperheight]{png/intersection_cube_sphere_mesh.png}
       \caption{\label{fig:corefinement}}
     \end{subfigure}
     \qquad
-    \begin{subfigure}[b]{0.5\textwidth}
+    \begin{subfigure}[b]{0.3\textwidth}
       \centering
-      \includegraphics[height=.2\paperheight]{png/cube_sphere_remeshed.png}
+      \includegraphics[height=.14\paperheight]{png/cube_sphere_remeshed.png}
       \caption{ \label{fig:corefinement_remeshed}}
     \end{subfigure}
-    \caption{\it (a) Union of the unit cube with an hyper ball. Notice
-      how the corefinement around the intersection results in badly shaped elements. (b) The remeshed \dealii{} Triangulation.}
+    \caption{\it (a) \texttt{Triangulation<3>} of a heart surface starting from an implicit function $f$. (b) Union of the unit cube with an hyper ball. Notice how the corefinement around the intersection gives badly shaped elements. (c) The remeshed \dealii{} Triangulation.}
```

Comment:
With three figures the rightmost one spills out into the margin a bit - we could probably fix that by trimming whitespace from the images.